public/styles/Spinner.scss

```diff
@@ -0,0 +1,18 @@
+.spinner-overlay {
+  background-color: $spinner-overlay;
+  color: $color-accent;
+  font-size: 1.3em;
+  height: 100%;
+  position: fixed;
+  text-align: center;
+  top: 0;
+  width: 100%;
+  z-index: 999999999999;
+
+  .fas {
+    -webkit-animation: fa-spin 1s infinite linear;
+    animation: fa-spin 1s infinite linear;
+    position: relative;
+    top: 45%;
```

Comment:
why are we manually adding the spin effect, font awesome already provides us with the spin animation. just add the class `fa-spin` to it

Reply:
Thanks for the feedback, I am well aware of the fa-spin class. I used this so that I can change the spin speed.